### PR TITLE
homework 9

### PR DESCRIPTION
**Pull request by artsiom-tsaryonau**

### CODE REVIEW

File: homework9/src/main/java/com/epam/lab/mentoring/task/Task4.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        System.out.println("Print adult (age >= 21) people only in consumer.");
+        people.forEach(p -> {
+            if (p.getAge() >= 21) {
+                System.out.println(p.getName() + " is adult");
+            };
+        });
+
+        List<String> names = Arrays.asList("John", "Pjotr", "Michelle");
+        System.out.println("Create adult people using supplier.");
+        List<Person> peopleGen = Stream.generate(() -> new Person(names.get(new Random().nextInt(3)), 21))
```
Comment (AShynkevich) [inline homework9/src/main/java/com/epam/lab/mentoring/task/Task4.java]: Limit value extract to constant

File: homework9/src/main/java/com/epam/lab/mentoring/task/TaskSupport.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.epam.lab.mentoring.task;
+
+public class TaskSupport {
+    public static <T> void printObject(T object) {
+        System.out.println(object.toString());
```
Comment (AShynkevich) [inline homework9/src/main/java/com/epam/lab/mentoring/task/TaskSupport.java]: Are you sure that you need to call "toString()" here?

File: homework9/src/main/java/com/epam/lab/mentoring/task/Task4.java (hunk 1 above)
Comment (artsiom-tsaryonau) [reply homework9/src/main/java/com/epam/lab/mentoring/task/Task4.java]: done

File: homework9/src/main/java/com/epam/lab/mentoring/task/TaskSupport.java (hunk 2 above)
Comment (artsiom-tsaryonau) [reply homework9/src/main/java/com/epam/lab/mentoring/task/TaskSupport.java]: removed

**Pull request merged**